game doesn't load

For some reasons game doesn't load. I use chromium browser on mac. I attached [an output from my console](http://pastebin.com/6ebVCQba)

I'm trying this with Chrome v53.0.2785.101 (64-bit) on Mac today. The game scene appeared, but no movement. And it was logging _"three.min.js:517 Error creating WebGL context."_ 

but I saw a few other people reporting this problem with ThreeJS and there's a workaround here: https://github.com/mrdoob/three.js/issues/404#issuecomment-20466146 _"Enter in the location bar chrome://flags and enable Override software rendering list. Then restart the browser. WebGL now should work."_

However that's probably a different problem. kharandziuk was getting errors with sound file URLs (last year)

--- a/server/server.js
+++ b/server/server.js
@@ -78,14 +78,18 @@ function Handler(req, res)
 	contentType = 'image/jpg';
 	break;
     }
+    var headers = {};
+    if (contentType)
+      headers['Content-Type'] = contentType;
+
     fs.exists(file, function(exists) {
 	if(exists) {
 	    fs.readFile(file,function(err,data) {
-		res.writeHead(200, {'Content-Type': contentType});
+		res.writeHead(200, headers);
 		res.end(data);
 	    });
 	} else {
-	    res.writeHead(404, {'Content-Type': contentType});
+	    res.writeHead(404, headers);
 	    res.end("Wizard killed the requested file with a Fireball! R.I.P "+file);
 	}
     });

